tests/igpu-container

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+set -eux
+
+# testflinger_queue: nvidia-jetson-agx-orin
+# testflinger_img_url: https://cdimage.ubuntu.com/nvidia-tegra/ubuntu-server/jammy/daily-preinstalled/current/jammy-preinstalled-server-arm64+tegra-igx.img.xz
+# testflinger_boot_media: usb
+
+# Details of the TF machine can be found at: https://certification.canonical.com/hardware/202306-31646/
+# Here is a setup guide: https://docs.google.com/document/d/1YhwbyWNGz4K8k8zsKhMBqbII5NkuD70cxCfOs5aPOl0/edit
+
+# Make sure that NVIDIA drivers are installed and are compatibles with the CUDA 12.5 version
+# Install CUDA Toolkit v12
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/arm64/cuda-keyring_1.1-1_all.deb
+sudo dpkg -i cuda-keyring_1.1-1_all.deb
```

Comment:
We generally assume root so sudo it probably not needed.

Reply:
fixed